MANUAL.md

```diff
@@ -603,45 +603,59 @@ Each timeslot has at least one recurrence by default. In the _Create New Timeslo
    <img src="public/screenshots/manual/Screenshot 2022-09-08 at 12.56.02.png"/>
 
 
-## Manage your contact details in Argus
-Add, edit or delete phone numbers in your settings. Phone numbers that are present in your settings can be used when [customizing alarm notifications](#customize-alarm-notifications-in-argus).
+## Manage your contact details (destinations) in Argus
+Add, edit or delete contact details, aka destinations, in your settings. Destinations that are present in your settings can be used when [customizing alarm notifications](#customize-alarm-notifications-in-argus).
 
-<img src="public/screenshots/manual/Screenshot 2022-09-08 at 14.00.29.png"/>
+<img src="public/screenshots/manual/Screenshot 2023-04-25 at 18.33.57.png"/>
 
-### Access your contact details in settings
+In Argus, _emails_ and _phone numbers_ are the destinations that are configured by default.
```

Comment:
Our included phone number support isn't exactly kosher. Dunno how to improve this doc on this point but we should probably stop including "sms-as-email" by default. Writing this in order to remember it.